crates/stages/src/sets.rs

```diff
@@ -71,8 +71,8 @@ use std::sync::Arc;
 /// - [`StorageHashingStage`]
 /// - [`MerkleStage`] (execute)
 /// - [`TransactionLookupStage`]
-/// - [`IndexStorageHistoryStage`]
-/// - [`IndexAccountHistoryStage`]
+/// - [`IndexStorageHistoriesStage`]
+/// - [`IndexAccountHistoriesStage`]
```

Comment:
IMO stage names should be left intact and singular as before, because we're indexing the **history** of account and storage changes, not **histories**.